### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by angkl0**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Ang Kuang Long
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kuanglong.png" width="200px">
```
Comment (parnikkapore) [inline docs/AboutUs.md:53]: The photo should be named after your GH username (see https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme)

**Review by parnikkapore — CHANGES_REQUESTED** [review]
Looking good so far. Just a change to the image file name is needed.

afaik we do need to resolve the conflicts on your branch.

File: README.md [hunk 2]
```diff
@@ -13,3 +13,25 @@
 * It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
 * For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
 * This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+
```
Comment (parnikkapore) [inline README.md:16]: Perhaps add to the features section as well?

File: docs/AboutUs.md (hunk 1 above)
Comment (angkl0) [reply docs/AboutUs.md]: Done

File: README.md (hunk 2 above)
Comment (angkl0) [reply README.md:16]: Done

**Review by parnikkapore — APPROVED** [review]
LGTM!

**Pull request merged**